.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.xsd]
+indent_style = tab
+insert_final_newline = false
+
+[.travis.yml]
+indent_size = 2
+
+[composer.json]
+indent_style = tab
```

Comment:
what if we change composer indent by spaces?

Reply:
I think I did that on the static-analysis branch.
I could change it to spaces on this branch if you'd prefer ?